modules/core/core-api/src/main/java/com/enonic/xp/node/DuplicateNodeParams.java

```diff
@@ -57,6 +63,16 @@ public Boolean getIncludeChildren()
         return includeChildren;
     }
 
+    public String getName()
+    {
+        return name;
+    }
+
+    public NodePath getParentPath()
```

Comment:
```suggestion
    public NodePath getParent()
```

same as in `CreateNodeParams`